### DISCUSSION TITLE
Decaying of mirror cache

### DISCUSSION THREAD

**Discussion by Geobert** [moriturus/ktra#14] [Ideas]
So we're now using Ktra in my company and it works well!

The guys asked me if there was some kind of decaying feature for the mirror functionnality so if a crate+version is not used for X time, delete it from the cache to avoid an ever increasing cache. 

I don't think there's such mechanism hence this feature request :)

**Comment by moriturus** [comment]
I'm so happy to hear that `ktra` is working well!

honestly, it is hard to make a decision to implement such a function into `ktra`.  
Because I feel surely that is convenient if it were but at the same time, I wonder if `ktra` should manage the storage.

Please let me think about that for a couple of days.

**Reply by Geobert** [reply]
Thank you for considering it :)
